### PR TITLE
Bump packages

### PR DESCRIPTION
**Pull request by timbonicus**
`version:bump` and related changes from minor revs

### CODE REVIEW

File: packages/app/src/App.tsx [hunk 1]
```diff
@@ -58,7 +66,13 @@ const routes = (
       path="/cost-insights/labeling-jobs"
       element={<CostInsightsLabelDataflowInstructionsPage />}
     />
-    <Route path="/docs" element={<TechdocsPage />} />
+    <Route path="/docs" element={<TechDocsIndexPage />}>
+      + <DefaultTechDocsHome />+{' '}
```
Comment (OrkoHunter) [inline packages/app/src/App.tsx:70]: This line looks a bit weird with the `+` 🤔

**Review by OrkoHunter — COMMENTED** [review]
Great stuff!

File: packages/app/src/App.tsx (hunk 1 above)
Comment (timbonicus) [reply packages/app/src/App.tsx]: Oh nice catch! Sloppy copy/paste from changelog diffs. Surprisingly the app booted up and seemed to work fine in manual testing 😮

Comment (OrkoHunter) [reply packages/app/src/App.tsx]: Yeah that's surprising!

File: packages/app/package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "@testing-library/user-event": "^12.0.7",
     "@types/jest": "^26.0.7",
     "@types/node": "^12.0.0",
-    "@types/react-dom": "^16.9.8",
+    "@types/react-dom": "*",
```
Comment (nissayeva) [inline packages/app/package.json:45]: why `*` here?

Comment (OrkoHunter) [reply packages/app/package.json:45]: Some discussions here https://github.com/backstage/backstage/pull/6697 :)

**Review by OrkoHunter — APPROVED** [review]

**Pull request merged**